packages/blade/src/components/Icons/_Svg/Svg/types.ts

```diff
@@ -6,4 +6,4 @@ export type SvgProps = {
   height: `${string}px`;
   viewBox: string;
   width: `${string}px`;
-} & StyledProps;
+} & Omit<StyledProps, 'order'>; // Order prop on SVG has different meaning so removing this prop from styled-props
```

Comment:
It'll be removed from the type but consumers can still pass it anyway right?

Reply:
yup. It will be treated as CSS `order` prop